preload/resources/preload_helper.js

```diff
@@ -14,7 +14,7 @@ function verifyNumberOfDownloads(url, number)
 {
     var numDownloads = 0;
     performance.getEntriesByName(getAbsoluteURL(url)).forEach(entry => {
-        if (entry.transferSize > 0) {
+        if (entry.transferSize === undefined || entry.transferSize > 0) {
```

Comment:
Sanity check: is transferSize even necessary? Is the intent to explicitly count number of resources fetched over the network, as opposed to fetched from HTTP cache?

Reply:
I'm *think* that was the original intent, but not 100% sure. @ddragana introduced it as part of https://github.com/web-platform-tests/wpt/commit/e7327c5ba23c244c1e98358f9622cd1e84202ba2

@ddragana - Can you confirm?